README.md

```diff
@@ -60,19 +67,19 @@ The specification below describes the `trigger` metadata in `ScaledObject` resou
   triggers:
     - type: external
       metadata:
-        scalerAddress: keda-cosmosdb-scaler.keda:4050 # Mandatory. Address of the external scaler service. 
-        connection: <connection>                      # Mandatory. Connection string of Cosmos DB account with monitored container.
-        databaseId: <database-id>                     # Mandatory. ID of Cosmos DB database containing monitored container.
-        containerId: <container-id>                   # Mandatory. ID of monitored container.
-        leaseConnection: <lease-connection>           # Mandatory. Connection string of Cosmos DB account with lease container.
-        leaseDatabaseId: <lease-database-id>          # Mandatory. ID of Cosmos DB database containing lease container.
-        leaseContainerId: <lease-container-id>        # Mandatory. ID of lease container.
-        processorName: <processor-name>               # Mandatory. Name of change-feed processor used by listener application.
+        scalerAddress: keda-external-scaler-azure-cosmos-db.keda:4050 # Mandatory. Address of the external scaler service.
+        connection: <connection>               # Mandatory. Connection string of Cosmos DB account with monitored container.
+        databaseId: <database-id>              # Mandatory. ID of Cosmos DB database containing monitored container.
+        containerId: <container-id>            # Mandatory. ID of monitored container.
+        leaseConnection: <lease-connection>    # Mandatory. Connection string of Cosmos DB account with lease container.
+        leaseDatabaseId: <lease-database-id>   # Mandatory. ID of Cosmos DB database containing lease container.
+        leaseContainerId: <lease-container-id> # Mandatory. ID of lease container.
+        processorName: <processor-name>        # Mandatory. Name of change-feed processor used by listener application.
 ```
 
 ### Parameter List
 
-- **`scalerAddress`** - Address of the external scaler service. This would be in format `<scaler-name>:<scaler-namespace>:<port>`. The port is hardcoded to be `4050` in the external scaler. If you haven't updated the `name` and `namespace` properties in file `deploy/deploy-scaler` when applying the configuration, the metadata value would be `keda-cosmosdb-scaler.keda:4050`.
+- **`scalerAddress`** - Address of the external scaler service. This would be in format `<scaler-name>:<scaler-namespace>:<port>`. The port is hardcoded to be `4050` in the external scaler. If you installed 'Cosmos DB scaler' Helm chart in `keda` namespace and did not specify custom values, the metadata value would be `keda-external-scaler-azure-cosmos-db.keda:4050`.
```

Comment:
What about this?
```suggestion
- **`scalerAddress`** - Address of the external scaler service. This would be in format `<scaler-name>.<scaler-namespace>:<port>`. The port is hardcoded to be `4050` in the external scaler.
  - If you installed the external scaler through Helm in the `keda` namespace with the defaults then the scaler address must be `keda-external-scaler-azure-cosmos-db.keda:4050`.
```

Also, I find this a bit confusing:
> The port is hardcoded to be `4050` in the external scaler

What do you mean here? Because the value is added in your example, or is this for non-Helm scenarios?

Reply:
The port for `Service` is configurable. I will fix the wording. The port for the container app is not configurable though. Inside the application, the GRPC port needs to be set while creating the host-builder. But fetching the port from app-settings required a host already created. Creating two hosts turned out to be messy, so I had to hardcode it for the container app.

Reply:
No problem at all, just wanted to clarify the port.